File: official/docs/php/current/batches/add-shipments.php
<?php

$client = new \EasyPost\EasyPostClient('EASYPOST_API_KEY');

$batch = $client->batch->addShipments(
    'batch_...',
    [
        'shipments' => [
            ['id' => 'shp_...'],
            ['id' => 'shp_...'],
        ]
    ]
);

echo $batch;

File: official/docs/python/current/batches/add-shipments.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

batch = client.batch.add_shipments(
    "batch_...",
    shipments=[
        {"id": "shp_..."},
        {"id": "shp_..."},
    ],
)

print(batch)
